README.md

```diff
@@ -25,11 +27,13 @@ We use Travis CI for testing - add a test if it makes sense, and make sure the b
 
 ## Deploy
 
-These docs are hosted on GitHub Pages. We do a couple things to make this nice:
-
-* Use the [gh-pages](https://github.com/tschaub/gh-pages) library for deploying
-* Use the [404 redirection trick](https://github.com/rafrex/spa-github-pages) to pass path/hash as query param to `/index.html`
+These docs are hosted on GitHub Pages. To deploy what you have locally:
 
 ```bash
 yarn run deploy
 ```
+
+Under the hood, we use a few tools to make this work:
+
+* Use the [gh-pages](https://github.com/tschaub/gh-pages) library for deploying
+* Use the [404 redirection trick](https://github.com/rafrex/spa-github-pages) to pass path/hash as query param to `/index.html`
```

Comment:
Maybe just prefix these two with "We" so it's "We use the..." to make it clear that it's not something the user has to do?